apps/core-app/src/main/modules/database/index.test.ts

```
@@ -0,0 +1,123 @@
+async function runWalCheckpoint(
+  module: DatabaseModule,
+  mode: 'PASSIVE' | 'TRUNCATE'
+): Promise<void> {
+  await (
+    module as unknown as {
+      runWalCheckpoint: (mode: 'PASSIVE' | 'TRUNCATE') => Promise<void>
+    }
+  ).runWalCheckpoint(mode)
+}
+
+describe('DatabaseModule WAL checkpoint maintenance', () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)